src/Sulu/Bundle/WebsiteBundle/Tests/Functional/BaseFunctional.php

```diff
@@ -60,14 +60,14 @@ public function setUp(): void
 
     protected function create(string $webspaceKey, array $data): AnalyticsInterface
     {
-        $entity = $this->setData($this->analyticsRepository->createNew(), $webspaceKey, $data);
+        $entity = $this->setAnalyticsData($this->analyticsRepository->createNew(), $webspaceKey, $data);
         $this->entityManager->persist($entity);
         $this->entityManager->flush();
 
         return $entity;
     }
 
-    protected function setData(AnalyticsInterface $analytics, string $webspaceKey, array $data): AnalyticsInterface
+    protected function setAnalyticsData(AnalyticsInterface $analytics, string $webspaceKey, array $data): AnalyticsInterface
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes